modules/apigee_edge_teams/src/Entity/ListBuilder/TeamListBuilder.php

```diff
@@ -110,4 +110,33 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function render() {
```

Comment:
Missing return typehint.

Reply:
This class is overwriting a method from `Drupal\Core\Entity\EntityListBuilder` which doesn't provide a return typehint, so it just follows the same signature.

Reply:
Drupal core will not provide return type hints for years, but I thought that this contrib module start adding/using those and parameter typehints consistenly because it makes the code type-safe and easier to be debugged. I agree the current codebase already contains 70% return typehint only, this is what I try to improve with this comment.